src/ansys/fluent/core/__init__.py

```diff
@@ -83,7 +84,11 @@ def version_info() -> str:
 # Whether stream and cache commands state
 DATAMODEL_USE_NOCOMMANDS_DIFF_STATE = True
 
+# Parent directory where codegen writes out API files
+GENERATED_API_DIR = (Path(__file__) / ".." / "generated").resolve()
```

Comment:
Rather than inside a `"generated"` directory, thoughts about a name that is more specific such as `"API_files"` or something similar? 

`ansys/fluent/core/API_files` vs `ansys/fluent/core/generated`

Reply:
I think the facts that they are generated and that they are API files are both salient pieces of information. I'm fine with the current name, but I would also have nothing against `generated_api`.

Reply:
For now, I won't change the name, will revisit the idea in the next refactoring.